### PR TITLE
refactor: move event util

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- move event util from /clickhouse to /posthog

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: posthog/demo/__init__.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def demo_route(request: Request):
     user.save()
     EventDefinition.objects.get_or_create(team=team, name="$pageview")
 
-    from ee.clickhouse.models.event import get_events_by_team
+    from posthog.models.event.util import get_events_by_team
```
Comment (neilkakkar) [inline posthog/demo/__init__.py:31]: these can probably move to top of file now

File: posthog/models/event/event.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 from django.db import models
 from django.utils import timezone
 
-from .team import Team
+from ..team import Team
```
Comment (neilkakkar) [inline posthog/models/event/event.py:11]: make this absolute import?

File: posthog/tasks/status_report.py [hunk 3]
```diff
@@ -69,7 +69,7 @@ def status_report(*, dry_run: bool = False) -> Dict[str, Any]:
             params = (team.id, report["period"]["start_inclusive"], report["period"]["end_inclusive"])
             team_report: Dict[str, Any] = {}
             # pull events stats from clickhouse
-            from ee.clickhouse.models.event import (
+            from posthog.models.event.util import (
```
Comment (neilkakkar) [inline posthog/tasks/status_report.py:72]: Hmm, some of these imports are at the top, others here. Worth moving all to the top now? (including the `posthog.models.person` ones right below

**Review by neilkakkar — APPROVED** [review]
minor nits!

**Comment by neilkakkar** [comment]
oh - although the failing test seems valid - a borked import

**Review by neilkakkar — CHANGES_REQUESTED** [review]
just switching so we don't miss the failed test!

**Comment by EDsCODE** [comment]
Merging because the import was fixed by the accompanying cloud PR

**Pull request merged**